src/util/color/colorpalette.cpp

```diff
@@ -0,0 +1,42 @@
+#include "colorpalette.h"
+
+namespace {
+constexpr mixxx::RgbColor kColorMixxxRed = mixxx::RgbColor(0xC50A08);
+constexpr mixxx::RgbColor kColorMixxxYellow = mixxx::RgbColor(0x32BE44);
+constexpr mixxx::RgbColor kColorMixxxGreen = mixxx::RgbColor(0x0044FF);
+constexpr mixxx::RgbColor kColorMixxxCeleste = mixxx::RgbColor(0xF8D200);
+constexpr mixxx::RgbColor kColorMixxxBlue = mixxx::RgbColor(0x42D4F4);
+constexpr mixxx::RgbColor kColorMixxxPurple = mixxx::RgbColor(0xAF00CC);
+constexpr mixxx::RgbColor kColorMixxxPink = mixxx::RgbColor(0xFCA6D7);
+constexpr mixxx::RgbColor kColorMixxxWhite = mixxx::RgbColor(0xF2F2FF);
```

Comment:
This constant should be made public and reused, because it is also needed for initializing `Cue`.

Reply:
You should use `ColorPaletteSettings.getHotcueColors()` for that.

Reply:
But 0xF2F2FF is used literally in `Cue::Cue()` -> violates DRY

Reply:
Fair point. Looks like there's no straightforward way to get the current hotcue color palette inside the Cue constructor.

Reply:
Done.